src/experiment/metrics/basicmetrics.jl

```diff
@@ -235,7 +238,7 @@ end
 plot the relative scores ( compared to the optimal ) of the heuristic during the search for fixed instances along the training. This plot is 
 meaningful only if the metrics is one from the evaluator (ie. the instance remains the same one).
 """
-function plotRewardVariation(metrics::BasicMetrics{TakeObjective, LearnedHeuristic{SR,R,A}}; filename::String="") where {SR, R, A}
+function plotRewardVariation(metrics::BasicMetrics{<:AbstractMetrics, LearnedHeuristic{SR,R,A}}; filename::String="") where {SR, R, A}
```

Comment:
We should be able to plot the reward evolution of a learned heuristic relatively to any kind of problem ( considering an Objective or not considering an Objective ).